docs/usage-examples/deleteOne.txt

```diff
@@ -0,0 +1,66 @@
+.. _laravel-delete-one-usage:
+
+=================
+Delete a Document
+=================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: delete one, remove, code example
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can delete a document in a collection by retrieving a single document and calling
+the ``delete()`` method on an Eloquent model or a query builder.
+
+Pass a query filter to the ``where()`` method, sort the matching documents, and call the
+``first()`` method to retrieve only the first document. Then, delete this matching document
+by calling the ``delete()`` method.
+
+Example
+-------
+
+This usage example performs the following actions:
+
+- Uses the ``Movie`` Eloquent model to represent the ``movies`` collection in the
+  ``sample_mflix`` database.
+- Deletes a document from the ``movies`` collection that matches a query filter.
```

Comment:
I don't think we should have a period on these since they're not full sentences (subject is in the intro)

```suggestion
- Uses the ``Movie`` Eloquent model to represent the ``movies`` collection in the
  ``sample_mflix`` database.
- Deletes a document from the ``movies`` collection that matches a query filter.
```

Reply:
Feel free to double-check me on this one - I'm not 100% sure if we still consider it a fragment with the subject implied in the intro

Reply:
Agreed - I think these are all fragments. Fixed!